### PR TITLE
Use credentials plugin for conduit

### LINKED ISSUE #56: Use credentials-plugin for conduit token/URL
https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Plugin

### PR DESCRIPTION
**Pull request by sectioneight**
This breaks the dependency between the Wrapper and the Notifier for sharing
credentials, and is generally a Much Better Way to handle sensitive data.

Fixes #56

### CODE REVIEW

File: src/main/java/com/uber/jenkins/phabricator/ConduitCredentialsDescriptor.java [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+import hudson.model.Job;
+import hudson.security.ACL;
+import hudson.util.ListBoxModel;
+import jenkins.model.Jenkins;
+import org.kohsuke.stapler.AncestorInPath;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ConduitCredentialsDescriptor {
```
Comment (sectioneight) [inline src/main/java/com/uber/jenkins/phabricator/ConduitCredentialsDescriptor.java]: Not in love with this static class thing

**Comment by sectioneight** [comment]
cc @cburroughs

**Comment by cburroughs** [comment]
Cool! I think there is some confluence wiki page you can do so it will show up under related olugin on https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Plugin

**Comment by sectioneight** [comment]
Yea, I was planning on adding that after cutting a release that actually includes the credentials functionality (which I'll do after getting the Harbormaster HTTP stuff reviewed/merged)

**Pull request merged**